code/modules/clothing/clothing.dm

```diff
@@ -304,21 +313,24 @@
 	return TRUE
 
 /obj/item/clothing/under/proc/set_sensors(mob/user)
-	if(!user.Adjacent(src) || !ishuman(user) || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED))
+	if(!user.Adjacent(src))
+		to_chat(user, "<span class='warning'>You have moved too far away!</span>")
```

Comment:
Message is wrong, also, add the other check back too

Reply:
Done.